versioned_docs/version-1.6/reference/manifest.mdx

```diff
@@ -271,6 +271,15 @@ deploy:
 
 If you define an image in the okteto manifest, `okteto deploy` will always run in remote mode.  If you don't define an image, and you run in remote mode, okteto will use a [default container image](https://hub.docker.com/r/okteto/installer).
 
+You can explicitly set the `remote` field in the `deploy` section without using the `--remote` flag:
```

Comment:
--remote is not available on 1.6, so let's remove it from this version of the docs. 
```suggestion
You can explicitly set the `remote` field in the `deploy` section without using the `--remote` flag:
```

Reply:
resolved!